packages/forms-web-app/src/views/includes/scripts.njk

```diff
@@ -1 +1,5 @@
 <!-- Javascript -->
+<script src="/assets/govuk/all.js"></script>
+<script>
+  window.GOVUKFrontend.initAll()
+</script>
```

Comment:
Not against doing this site wide, but if so we need to remove the per page approach that is currently in use:

https://github.com/Planning-Inspectorate/appeal-planning-decision/blob/master/packages/forms-web-app/src/views/appellant-submission/site-access.njk#L93

https://github.com/Planning-Inspectorate/appeal-planning-decision/blob/master/packages/forms-web-app/src/views/appellant-submission/site-access-safety.njk#L109

Reply:
Didn't realise this was being done per page. The reason for site wide is missing out on multiple small things around error summaries and the like